elastic/kibana
# [ObsUX][Synthtrace] Replace `multistream` use with own util method

## Summary

A whole dependency was being pulled in for doing something that could easily be done just using Node.js own utils and be made into a simple method. As such, `multistream` has been removed since there is no other place in the codebase that is using it.

## How to test

* Load Kibana local dev environment using synthtrace data/scenarios. No scenario should fail to load normally, and all tests using synthtrace data should pass as expected.

Closes #203860
